src/backups.py

```diff
@@ -161,6 +158,11 @@ def _retrieve_s3_parameters(self) -> Tuple[Dict, List[str]]:
                 f"Missing required S3 parameters in relation with S3 integrator: {missing_parameters}"
             )
 
+        # Add some sensible defaults (as expected by the code) for missing optional parameters
+        s3_parameters.setdefault("endpoint")
```

Comment:
please
```suggestion
        s3_parameters.setdefault("endpoint", "https://s3.amazonaws.com")
```

Reply:
addressed in 2f1db018fed3b71ac85361e7b9ca380e0cc7c95a